v2/src/components/appInfoForm/style.css

```diff
@@ -157,15 +157,22 @@
 }
 
 .app-info-form-field-input {
-  border: 1px solid rgb(72, 72, 72);
+  border: 1px solid rgb(69, 69, 69);
+  background-color: rgb(0, 0, 0);
+  color: rgb(255, 255, 255);
   box-shadow: 0px 0px 0px 0px transparent;
   outline: none;
   transition: border .15s, box-shadow .15s;
+  padding: 8px 20px;
+  font-family: Roboto;
 }
 
-.app-info-form-field-input:focus {
-  border: 1px solid rgb(255, 128, 0);
-  box-shadow: 0px 0px 0px 2px rgb(255, 191, 128);
+.app-info-form-field-input:focus {  
+  box-shadow: 0px 0px 3px 1px rgb(255, 153, 51, 0.8);
```

Comment:
The Design use 0.5px solid border. But browser will treat 0.5px solid border as a 1px solid.
Therefore, it looks too much glowing, unlike the design.

So, I remove it. And it looks more similar to the design.

![appInfoform color](https://user-images.githubusercontent.com/13642906/188363510-a6cf0673-0b75-42f1-871e-ad0e55b65f2b.gif)

Reply:
Have you spoken to @mufassirkazi about this?

Reply:
Feedback from mufassir:
- The border for focus can be 1px
- When focussing on an input field the placeholder value should dissapear

Reply:
Next time I'll ask Mufassir if there is case like this. Meanwhile, its updated.